core/src/models/BibleReference.ts

```diff
@@ -87,6 +87,7 @@ export interface IBibleReferenceRangeQuery extends IBibleReferenceRange {
     versionUid: string;
     // TODO: implement
     targetVersionUid?: string;
+    skipPartialSectionsInDocument?: boolean;
 }
 
 /* types / interfaces neded for bible-passage-reference-parser */
```

Comment:
neded => needed

Reply:
done

Reply:
i changed then attribute to `skipPartialWrappingSectionsInDocument` and added a comment